catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -729,7 +730,7 @@ public Response updateDocument(
         CreateInfo createInfo = null;
         if (multipartBody != null) {
           List<Attachment> contentParts = multipartBody.getAllAttachments();
-          if (contentParts != null && contentParts.size() > 0) {
+          if (contentParts != null && !contentParts.isEmpty()) {
```

Comment:
✏️ Apache commons is already present; can replace with `if (CollectionUtils.isNotEmpty(contentParts))`